fabric/bin/peer.sh

```diff
@@ -331,25 +331,39 @@ handle_channel_join() {
     try $RUN ${FABRIC_BIN_DIR}/peer "${ARGS_EXEC[@]}"
 
     # - handle ercc
-    say "Installing & Instantiating ercc on channel '${CHAN_ID}' ..."
-    #   - install ercc
-    try $RUN ${FABRIC_BIN_DIR}/peer chaincode install -n ${ERCC_ID} -v ${ERCC_VERSION} -p github.com/hyperledger-labs/fabric-private-chaincode/ercc/cmd
-    sleep 1
-    #   - instantiate ercc iff "channel creation" peer
-    if [ -e "${FABRIC_STATE_DIR}/${CHAN_ID}.creator" ]; then
-	try $RUN ${FABRIC_BIN_DIR}/peer chaincode instantiate -n ${ERCC_ID} -v ${ERCC_VERSION} -c '{"args":["init"]}' -C ${CHAN_ID} -V ercc-vscc
-	sleep 4
-	try rm "${FABRIC_STATE_DIR}/${CHAN_ID}.creator"
-    fi
-    #   - get SPID (mostly as debug output)
-    try $RUN ${FABRIC_BIN_DIR}/peer chaincode query -n ${ERCC_ID} -c '{"args":["getSPID"]}' -C ${CHAN_ID}
+    say "Installing ercc on channel '${CHAN_ID}' ..."
+    echo "Packaging chaincode ..."
+    try $RUN ${FABRIC_BIN_DIR}/peer lifecycle chaincode package ercc.tar.gz -p ../../ercc --lang golang --label erccv1 
```

Comment:
can we create the package in some tempspace rather than inside the repo? If in repo, then we definitely should remove it after install and/or errors (probably a good idea even if we install in a temp dir).  Maybe then also define a variable so we have the path only at a single place?